OpenRA.Game/Traits/Player/PlayerColorPalette.cs

```diff
@@ -17,13 +17,17 @@ namespace OpenRA.Traits
 	public class PlayerColorPaletteInfo : ITraitInfo
 	{
 		[Desc("The name of the palette to base off.")]
-		public readonly string BasePalette = null;
+		[PaletteReference] public readonly string BasePalette = null;
+
 		[Desc("The prefix for the resulting player palettes")]
-		public readonly string BaseName = "player";
+		[PaletteDefinition] public readonly string BaseName = "player";
```

Comment:
This is a player palette, so the BaseName by itself is not a valid palette name.